insert.md

```diff
@@ -165,38 +165,127 @@ If you don't list column names, the statement will use the columns of the table
 ~~~
 
 ### Insert and Return Values
-
+  
+In this example, the `RETURNING` clause returns the `id` and `balance` of the rows inserted. The language-specific versions assume that you have installed the relevant [client drivers](install-client-drivers.html). 
```

Comment:
I miss the remark here from Ben in #766, about the fact that most drivers propose both an Execute and a Query interface. Since we cannot (or at least will not in the short term) provide examples in all languages, we can share this piece of general wisdom with the user: "if you're used to run insert via your language's / driver's `exec` method, use it with `query` instead and RETURNING". Or something like that.